src/FilterService.Mappers/Models/UserInfoMapper.cs

```diff
@@ -28,14 +28,10 @@ public class UserInfoMapper : IUserInfoMapper
       List<ProjectInfo> projectsInfo,
       List<ProjectData> projectsData)
     {
-      if (usersData is null)
-      {
-        return null;
-      }
-
-      return usersData.Select(x => new UserInfo
+      return usersData?.Select(x => new UserInfo
       {
         Id = x.Id,
+        AvatarId = x.ImageId,

```

Comment:
AvatarId = x?.ImageId
and please, move to line 38, after MiddleName

Reply:
userData is known to be not null